src/main/java/org/mule/extension/http/internal/request/HttpPollingSource.java

```diff
@@ -155,7 +155,8 @@ public void success(Result<InputStream, HttpResponseAttributes> result) {
 
           @Override
           public void error(Throwable throwable) {
-            LOGGER.error("There was an error in HTTP Polling Source of uri '{}'", resolvedUri, throwable);
+            LOGGER.error("There was an error in HTTP Polling Source of uri '{}'", resolvedUri);
```

Comment:
IMO, this type of separation makes troubleshooting and readability harder, specially when logs get ingested by third party tools

Reply:
I separated it because Rodro suggested that if not, log4j might not shot it as an error. If it works, I leave it as my previous attempt